thewalrus/_hafnian.py

```diff
@@ -14,23 +14,21 @@
 """
 Hafnian Python interface
 """
-
+import warnings
 from functools import lru_cache
 from collections import Counter
 from itertools import chain, combinations
 import numba
-
 import numpy as np
+from sympy import true
```

Comment:
You don't need this import

Reply:
Yeah, I'm not sure why this import got added. I'm going to remove it.